lib/order.rb

```diff
@@ -0,0 +1,19 @@
+
+class Order
+
+  attr_reader:order
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.